app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt

```diff
@@ -0,0 +1,41 @@
+package org.covidwatch.android
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.navigation.findNavController
+import org.covidwatch.android.databinding.FragmentSplashBinding
```

Comment:
I know in the other PR @Apisov mentioned not putting template code in the Fragment. I'm okay with for now, but what are the alternatives?

Reply:
What template code do you refer to - I think I removed everything.

Reply:
Setting the `onClickListener`. Also to be clear idk if there's an alternative/this falls in the exact same category

Reply:
That's required wiring for the button to lead to the next screen. 
What Pavlo was refering to were the unused `Intent` parameters that come with a new Fragment template and were left by whoever initially created it.